documentation/how-to/digitize.en.md

```diff
@@ -145,6 +145,19 @@ QField allows you to merge features and their geometries into a single feature.
 features on the map, select them in the features list and merge them by selecting
 the *Merge Selected Features* in the *Menu (⁝) Button*.
 
+To execute polygon merging within QField, ensure the following conditions:
```

Comment:
It's not only polygon merging, it's all geometry types

Reply:
Great, I will fix it